src/DuckNet.cpp

```diff
@@ -248,7 +248,7 @@ int DuckNet::setupWebServer(bool createCaptivePortal, String html) {
   });
 
   // Captive Portal form submission
-  webServer.on("/formSubmit.json", HTTP_POST, [&](AsyncWebServerRequest* request) {
```

Comment:
I believe you need to revert this. Please test submitting on captive portal to see if it still works properly

Reply:
Maybe try pulling from CDP master?